src/core_codemods/break_or_continue_out_of_loop.py

```diff
@@ -0,0 +1,63 @@
+from typing import Union
+
+import libcst as cst
+
+from codemodder.codemods.base_codemod import Metadata, ReviewGuidance
+from codemodder.codemods.libcst_transformer import (
+    LibcstResultTransformer,
+    LibcstTransformerPipeline,
+)
+from codemodder.codemods.utils_mixin import AncestorPatternsMixin
+from codemodder.codetf import Reference
+from core_codemods.api.core_codemod import CoreCodemod
+
+
+class BreakOrContinueOutOfLoopTransformer(
+    LibcstResultTransformer, AncestorPatternsMixin
+):
+
+    change_description = "Removed break or continue statement out of loop."
+
+    def _handle_break_or_continue(
+        self,
+        original_node: cst.Break | cst.Continue,
+        updated_node: cst.Break | cst.Continue,
+    ):
+        ancestors = self.path_to_root(original_node)
+        # is it inside a for or while ?
+        if any(filter(lambda n: isinstance(n, cst.For | cst.While), ancestors)):
```

Comment:
I would just remove the commend and move this check to a `_is_inside_loop` method, for clarity